OSDOCS-14387:RN for localnet (#94131)
openshift/openshift-docs: release_notes/ocp-4-19-release-notes.adoc

==== Enabling DPU device management with the DPU Operator
With this release, {product-title} introduces the Data Processing Unit (DPU) Operator, enabling management of DPU devices. The DPU Operator manages components on compute nodes with DPUs, enabling the offloading of data workloads such as networking, storage, and security. This leads to improved cluster performance, reduced latency, and enhanced security, contributing to a more efficient infrastructure. For more information, see xref:../networking/networking_operators/dpu-operator/about-dpu.adoc#about-dpu-and-dpu-operator[About DPU and the DPU Operator].

[id="ocp-4-19-cluster-user-defined-networks-localnet_{context}"]

> ==== `Localnet` topology for user-defined networks (Generally Available)

Reviewer: ```suggestion
==== Localnet topology for user-defined networks (Generally Available)
```

> Administrators can now use the `ClusterUserDefinedNetwork` custom resource to deploy secondary networks on a `Localnet` topology that overlay network to your physical network. This feature provides pods and virtual machines attached to the overlay network (`Localnet` network) the ability to egress to the physical network. For more information, see xref:../networking/multiple_networks/primary_networks/about-user-defined-networks.adoc#nw-cudn-localnet[Creating a ClusterUserDefinedNetwork CR for a Localnet topology].

Reviewer: Can we check whether it's Localnet or localnet? I see both uses in the docs. I think in UDN we've been saying "Localnet", but it's "localnet" in other places. 

```suggestion
Administrators can now use the `ClusterUserDefinedNetwork` custom resource to deploy secondary networks on a `localnet` topology.  This feature enables pods and virtual machines connected to the Localnet network to egress to the physical network. For more information, see xref:../networking/multiple_networks/primary_networks/about-user-defined-networks.adoc#nw-cudn-localnet[Creating a ClusterUserDefinedNetwork CR for a Localnet topology].
```
